Resources/Prototypes/Entities/Clothing/Earpieces/ears.yml

```diff
@@ -1,16 +1,255 @@
+## Earpiece Headsets
+
 - type: entity
   parent: Clothing
-  id: RadioHeadsetEars
-  name: headset radio
+  id: HeadsetRadioEars
+  name: headset
   description: The radio to keep up to date in real time with fun onboard station activities
   components:
+  - type: Headset
   - type: Sprite
-    sprite: Clothing/Earpieces/headset.rsi
+    sprite: Objects/Devices/communication.rsi
     state: headset
   - type: Icon
-    sprite: Clothing/Earpieces/headset.rsi
+    sprite: Objects/Devices/communication.rsi
     state: headset
   - type: Clothing
     Slots:
     - ears
-    sprite: Clothing/Earpieces/headset.rsi
+    sprite: Objects/Devices/communication.rsi
+
+- type: entity
+  parent: Clothing
+  id: HeadsetRadioCargoEars
+  name: cargo headset
+  description:
+  components:
+  - type: Headset
+  - type: Sprite
+    sprite: Objects/Devices/communication.rsi
+    state: cargo_headset
+  - type: Icon
+    sprite: Objects/Devices/communication.rsi
+    state: cargo_headset
+  - type: Clothing
+    Slots:
+    - ears
+    sprite: Objects/Devices/communication.rsi
```

Comment:
You could make an abstract EarsRadio prototype for these and have all of them inherit from it to save some lines.